jupyter_scheduler/models.py

```diff
@@ -160,24 +160,46 @@ class DeleteJob(BaseModel):
     job_id: str
 
 
-class CreateJobDefinition(CreateJob):
+class CreateJobDefinition(BaseModel):
+    input_uri: str
+    output_prefix: str
+    runtime_environment_name: str
+    runtime_environment_parameters: Optional[Dict[str, EnvironmentParameterValues]]
+    output_formats: Optional[List[str]] = None
+    idempotency_token: Optional[str] = None
```

Comment:
Not sure the idempotency token makes sense for a job definition. Thoughts?

Reply:
Agree, it doesn't make sense to apply the same value on a recurring basis. We might need to revisit if users ask for a more sophisticated use of this feature.